updates/version.yaml

```diff
@@ -58,4 +58,6 @@
 4.6.1:
     - "Fix overwritten sitemap and the canonical tag is always added with the url of the current page"
 4.7.0:
-    - "Add Schema component"
\ No newline at end of file
+    - "Add Schema component"
+4.7.1:
+    - "Fix Schema syntax error"
```

Comment:
since your work was about favicon and fixing schema syntax was just one line fix, maybe you should consider naming version with something more accurate, like: 'Fix favicon'